src/main.c

```diff
@@ -111,10 +114,11 @@ static void movement_phase(bool do_run)
 }
 
 /**
- * @brief Configure the goal for the search phase.
+ * @brief Let the user configure the goal for the search phase.
```

Comment:
Adding configure_speed(false), this function it is also moving the robot on exploration phase. The same that for previous function, I think that the name and comments are unclear and is hard to follow the code.

Reply:
[Reply on a different comment]